mailpoet/lib/Config/Renderer.php

```diff
@@ -78,7 +79,8 @@ public function setupGlobalVariables() {
       'assets_url' => Env::$assetsUrl,
       'assets_manifest_js' => $this->assetsManifestJs,
       'assets_manifest_css' => $this->assetsManifestCss,
-    ]));
+    ],
+      WPFunctions::get()));
```

Comment:
Just a nitpick, but the indentation here looks a bit weird, and it took me a bit to understand which method was receiving `WPFunctions::get()` as a parameter. Maybe adding new lines and separating this line (https://github.com/mailpoet/mailpoet/pull/4588/commits/8750e88ebe8f0b77f69b42ac74891399943e1998#diff-54255286a5807b0526ab1fbe775608c4416a08fae0b4a1182bf0686855a1eb24R77) into three and maybe the corresponding change to https://github.com/mailpoet/mailpoet/pull/4588/commits/8750e88ebe8f0b77f69b42ac74891399943e1998#diff-54255286a5807b0526ab1fbe775608c4416a08fae0b4a1182bf0686855a1eb24R83 will help?

Reply:
Thanks. Addressed in [ba1d8e6](https://github.com/mailpoet/mailpoet/pull/4588/commits/ba1d8e62c973513591e9498a00a79dea8b8f5a38)